src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -289,10 +294,26 @@ export default {
 			return t('spreed', 'Lower hand (R)')
 		},
 
+		conversation() {
+			return this.$store.getters.conversation(this.token) || this.$store.getters.dummyConversation
+		},
+
+		isAudioAllowed() {
+			return this.conversation.publishingPermissions === PARTICIPANT.PUBLISHING_PERMISSIONS.ALL
```

Comment:
All is a binary flag list. So is publishingPermissions.
I guess a bit comparison for the respective flag makes more sense and ensures it works in the future if we ever split it?
```suggestion
			return this.conversation.publishingPermissions & PARTICIPANT.PUBLISHING_PERMISSIONS.AUDIO
```

Reply:
I did not check against the specific flags because for now the UI treats the permissions as a whole, all or nothing, so checking against all ensures that the whole UI behaviour would be consistent. I think that should be changed only once the rest of the UI is updated to handle specific flags too.